Civi/Api4/Generic/Traits/ArrayQueryActionTrait.php

```diff
@@ -104,6 +104,8 @@ private function filterCompare($row, $condition) {
     $value = $row[$condition[0]] ?? NULL;
     $operator = $condition[1];
     $expected = $condition[2] ?? NULL;
+    $isExpression = $condition[3] ?? FALSE;
+    // TODO: should we just give up if $isExpression === TRUE?
```

Comment:
Not sure about this.

Reply:
See comment above. We can revert changes to this file.